frontend/src/components/RelatedCourses.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div v-if="relatedCourses.data?.length" class="mt-10">
+        <div class="flex items-center justify-between mb-6">
```

Comment:
Please install pre-commit and ensure that all your files are formatted. You can run pre-commit run --all-files and then commit the formatted changes.